notebooks/00_HubQuickStart.ipynb

```diff
@@ -1,10 +1,67 @@
 {
```

Comment:
<p>This is clearly problematic. It needs to show the data frame for the commentary to make sense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass-demo/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>